lib/pliny/middleware/rescue_errors.rb

```diff
@@ -8,25 +8,12 @@ def initialize(app, options = {})
     def call(env)
       @app.call(env)
     rescue Pliny::Errors::Error => e
-      # blank out this field so that the error is not picked up by Rollbar
-      env["sinatra.error"] = nil
```

Comment:
Could we log this error, or at least that we are rescuing it? It might be handy to track down user-reported errors.